lib/std/atomic/int.zig

```diff
@@ -3,6 +3,9 @@
 // This file is part of [zig](https://ziglang.org/), which is MIT licensed.
 // The MIT license requires this copyright notice to be included in all copies
 // and substantial portions of the software.
+
+const builtin = @import("std").builtin;
+
 /// Thread-safe, lock-free integer
 pub fn Int(comptime T: type) type {
     return struct {
```

Comment:
One other thing to think about is if this should be an `extern struct` or `packed` struct so that it can be used inside of public ABIs.

Reply:
I'm leaning on `extern` as that preserves the alignment of `T` and can be part of extern structures/fns.